src/main/java/sc/fiji/bdvpg/bdv/projector/AccumulateMixedProjectorARGB.java

```diff
@@ -80,16 +79,14 @@ protected void accumulate(
 				continue;
 			}
 
-			String projectionMode = getProjectionMode( sourceIndex, sourceAndConverterToMetadata );
-
-			if ( projectionMode.equals( "Sum" ) )
+			if ( projectionModes[sourceIndex].equals( Projection.PROJECTION_MODE_SUM ) )
 			{
 				aAccu += a;
 				rAccu += r;
 				gAccu += g;
 				bAccu += b;
 			}
-			else if ( projectionMode.equals( "Avg" ))
+			else if ( projectionModes[sourceIndex].equals( Projection.PROJECTION_MODE_AVG ))
```

Comment:
same as previous

Reply:
I would leave it as a string for now unless we feel performance issues.
See my answer below.